src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -38,17 +38,22 @@ class ExpressionValidator extends ConstraintValidator
     private $expressionLanguage;
 
     /**
-     * @param PropertyAccessorInterface|null $propertyAccessor Optional as of Symfony 2.5
+     * @param PropertyAccessorInterface|null $propertyAccessor   Optional as of Symfony 2.5
```

Comment:
the comment ``Optional as of Symfony 2.5`` does not bring any value and should be removed IMO. when looking at the phpdoc, we don't really care about which version made it optional. Someone wanting to know this will look at the changelog or the git history